### DISCUSSION TITLE
Override AdaptiveColor, Force Dark/Light mode

### DISCUSSION THREAD

**Discussion by bensadeh** [charmbracelet/lipgloss#90] [Q&A] [answered]
Hey Team Charm,

I am currently using `lipgloss.AdaptiveColor{}` to adapt the colors of my TUI based on the user's color scheme. It is working great and I find this feature very useful. 

I was wondering if there was I way where I could override which color `AdaptiveColor` resolves to, so that a user could run the program like so: `cli --force-light-mode`.

I can see this being useful where getting the color scheme is not possible, like from `tmux` or `screen`. It can also be useful where the user's color scheme borders between the cutoff point between dark and light.

**Comment by muesli** [comment, marked as answer]
You can call `lipgloss.SetHasDarkBackground(true)` to override the color check:

https://github.com/charmbracelet/lipgloss/blob/master/color.go#L84

**Comment by bensadeh** [comment]
That's... exactly what I was looking for! Thanks for the quick reply 😊
